daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-features-concepts.md

```diff
@@ -323,6 +322,15 @@ fetch('https://postman-echo.com/get')
 
 {{% /codetab %}}
 
+{{% codetab %}}
+
+```go
+// DON'T DO THIS!
+
```

Comment:
@mikeee does this section and the 'do this' section on L373 need examples/content? or does it not apply for Go?